cds_ils/config.py

```diff
@@ -161,6 +161,12 @@ def _parse_env_bool(var_name, default=None):
 #: Scheduled tasks configuration (aka cronjobs).
 CELERY_BEAT_SCHEDULE = {
     **ILS_CELERY_BEAT_SCHEDULE,  # Parent config
+    # overwrites ILS config
+    "send_overdue_loan_reminders": {
+        "task": "invenio_app_ils.circulation.mail.tasks.send_overdue_loans_mail_reminder",  # noqa
```

Comment:
In the code of [this task](https://github.com/inveniosoftware/invenio-app-ils/blob/27e56c4cf482d4ebbd17b4fddd960646ee2b6235/invenio_app_ils/circulation/mail/tasks.py#L68) we are:

1. getting the config `ILS_CIRCULATION_MAIL_OVERDUE_REMINDER_INTERVAL` -> `3 days` by default.
2. getting all overdue loans
3. checking how many days have passed, e.g. `10`
4. calculating the `mod`, `10 % 3`
5. if `mod` is `0`, then send email

It is important to keep the cron every day, otherwise we will skip many of them.
You might want to tune the number of days via the config var.
Can  you please double check?